src/lib.rs

```diff
@@ -27,7 +27,7 @@ mod generated {
 }
 
 fn panic_hook(info: &std::panic::PanicInfo) {
-    panic!("{}", info);
+    eprint!("Panic from mozjs: {}", info);
```

Comment:
@jdm @sagudev While I try to update rust-mozjs, `test-panic` failed with 

```
process didn't exit successfully (signal: 4, SIGILL: illegal instruction)
```

and I realized the SIGILL is caused by panic in panic which is related to this line.

After changing the `panic` to `eprint` in this panic hook, I can pass the `test-panic` in rust-mozjs. Do you think it's fine to just `eprint` here instead of `panic` in mozjs?

Reply:
What does stderr is saying?

Reply:
Yep, eprint seems reasonable. Hopefully if the panic hook doesn't do anything special the rust continues handling the original panic oncethe handler returns.

Reply:
> What is stderr saying?

I didn't get any stderr and only get the SIGILL when the program panicked in a panic.

> Yep, eprint seems reasonable. Hopefully if the panic hook doesn't do anything special then rust continues handling the original panic once the handler returns.

Based on the `test-panic` test in rust-mozjs, I think rust will continue handling the original panic after the hook is done. because I can get the test pass after I made this line as eprint.

https://github.com/servo/rust-mozjs/blob/6dad062b9386e75165d258078a772d7ad53e9837/tests/panic.rs#L51-L61